src/TokenizedStrategy.sol

```diff
@@ -780,6 +780,17 @@ contract TokenizedStrategy {
         return _maxWithdraw(_strategyStorage(), owner);
     }
 
+    /**
+     * @notice Accepts a `maxLoss` variable in order to match the multi
+     * strategy vaults ABI.
+     */
+    function maxWithdraw(
+        address owner,
```

Comment:
Maybe a dumb question ... is there a reason to support strategy implementations from choosing to handle this?